File: 백준/Bronze/3003. 킹， 퀸， 룩， 비숍， 나이트， 폰/킹， 퀸， 룩， 비숍， 나이트， 폰.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        int[] requiredPieces = {1, 1, 2, 2, 2, 8};
        int[] foundPieces = new int[6];
        
        for (int i = 0; i < 6; i++) {
            foundPieces[i] = scanner.nextInt();
        }
        
        for (int i = 0; i < 6; i++) {
            int diff = requiredPieces[i] - foundPieces[i];
            System.out.print(diff + " ");
        }

        scanner.close();
    }
}

File: 백준/Bronze/3003. 킹， 퀸， 룩， 비숍， 나이트， 폰/킹， 퀸， 룩， 비숍， 나이트， 폰.cc
#include <iostream>
using namespace std;

int main() {
    int requiredPieces[] = {1,1,2,2,2,8};
    int inputPieces[6];
    for (int i = 0; i < 6; i++) {
        cin >> inputPieces[i];
    }
    for (int i = 0; i < 6; i++) {
        int diff = requiredPieces[i] - inputPieces[i];
        cout << diff << " ";
    }
    return 0;
}